lib/Mail.php

```diff
@@ -623,7 +623,13 @@ public function sendWithoutRendering(\Swift_Mailer $mailer = null)
 
         if ($event->hasArgument('mailer')) {
             $mailer = $event->getArgument('mailer');
-            $mailer->send($this);
+            try {
+                $mailer->send($this, $emailaddress);            
+            } catch (\Exception $e){
+                $mailer->getTransport()->stop();
+                sleep(10);
```

Comment:
Why `sleep()`?

Reply:
To make sure the transportation connection is really closed, otherwise we will run in occupied errors, Sleep() happen only in an error case. But I read a bit more about swiftmailer in the meantime, maybe the is a better solution, but this have to wait for the weekend.

Reply:
Ok, thanks, then I'll wait until you come back on this topic.